package.json

```diff
@@ -17,7 +17,7 @@
     "i18next": "^20.3.2",
     "lodash": "^4.17.21",
     "moment": "^2.29.1",
-    "node-sass": "^4.14.1",
```

Comment:
There are some breaking changes with node-sass 6, we can test it really well before merging

Reply:
Lets try and use sass as some are facing issues with node-sass compatibility with node versions

Reply:
@jacksonjp 
Thanks for working on this 😄 
3e6248f